packages/components/stamp/src/stamp.tsx

```diff
@@ -77,7 +71,7 @@ const tonesPropsMap = (
       borderColor: designTokens.colorPrimary85,
       color: designTokens.colorSuccess40,
     },
-    iconColor: themedValue('primary', 'success'),
+    iconColor: 'primary',
```

Comment:
It seems this value be `success`; that's what we had when we were using the theming hook.

Currently we can see the icon not rendering in the expected color:

![Screenshot 2024-04-23 at 18 24 11](https://github.com/commercetools/ui-kit/assets/97907068/28cc58ca-f18f-4974-8546-b9c07e7495d4)

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2785/commits/6ba27608f57b6f0121caaf6d2293725bca63a576

Thank you